src/CMakeLists.txt

```diff
@@ -24,7 +24,16 @@ ov_tokenizers_set_cxx_standard()
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-option(BUILD_FAST_TOKENIZERS OFF)
+get_target_property(OPENVINO_RUNTIME_COMPILE_DEFINITIONS openvino::runtime INTERFACE_COMPILE_DEFINITIONS)
+if("_GLIBCXX_USE_CXX11_ABI=0" IN_LIST OPENVINO_RUNTIME_COMPILE_DEFINITIONS)
```

Comment:
to migrate to https://github.com/openvinotoolkit/openvino/pull/24657 once that it's merged